sktime/utils/validation/forecasting.py

```diff
@@ -328,13 +328,24 @@ def check_scoring(scoring):
     ----------
     TypeError
         if object is not callable from current scope.
+    NotImplementedError
+        if metric requires y_pred_benchmark to be passed
     """
     # Note symmetric=True is default arg for MeanAbsolutePercentageError
     from sktime.performance_metrics.forecasting import MeanAbsolutePercentageError
 
     if scoring is None:
         return MeanAbsolutePercentageError()
 
+    if scoring._tags["requires_y_pred_benchmark"]:
```

Comment:
I would also add a kwarg, i.e. `check_scoring(..., allow_y_pred_benchmark=False)` and add that to the if condition.

Reply:
Sounds good. Is that so we can just toggle on the ability to allow those metrics later?

Reply:
Yes, and people may want to use `check_scoring` in other contexts I guess